modules/UI/videolayout/FilmStrip.js

```diff
@@ -66,13 +64,52 @@ const FilmStrip = {
             - parseInt(this.filmStrip.css('paddingRight'), 10);
     },
 
+    calculateThumbnailSize() {
+        let availableSizes = this.calculateAvailableSize();
+        let width = availableSizes.availableWidth;
+        let height = availableSizes.availableHeight;
+
+        return this.calculateThumbnailSizeFromAvailable(width, height);
+    },
+
     /**
-     * Calculates the thumbnail size.
+     * Normalizes local and remote thumbnail ratios
      */
-     calculateThumbnailSize () {
-        let availableHeight = interfaceConfig.FILM_STRIP_MAX_HEIGHT;
+     normalizeThumbnailRatio () {
```

Comment:
Here we recalculate proportions for local and remote videos in such a way that all of them will be measured with common units (i.e. if we have __1:1__ for remote video and __16:9__ for local one this method will normalize this values to __9:9__ and __16:9__ - in other words we're finding common denominator for the rational). It allows us to sum up ratios for width, for example, define available width for the current window and calculate then number of pixels available for one unit. So, if I'll continue previous example then let's define 15px as a number of pixels for one unit. Hence, we can easily calculate width for remote (__15 * 9 = 135px__) and for local one (__15 * 16 = 240px__).